slimserver.pl

```diff
@@ -778,6 +780,7 @@ sub showUsage {
     --httpport       => Activate the web interface on the specified port.
                         Set to 0 in order disable the web server.
     --httpaddr       => Activate the web interface on the specified IP address.
+    --advertiseaddr  => IP address to send to players via mysqueezebox.com.
```

Comment:
If you moved to `serverAddr()`, then you could broaden the description, too. It's basically the user facing IP address.